### PR TITLE
Changes to readme, additional md files/diagram

### PR DESCRIPTION
**Pull request by angelazhangmsft**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+* Want to know if we're planning on building a particular feature? File an issue
+* Got a great idea for a new utility or feature? File an issue/request/idea
+* Don't understand how to do something? File an issue/Community Guidance Request
+* Found an existing issue that describes yours? Great - upvote and add additional commentary / info / repro-steps / etc.
+
+### Complete the template
+
+**Please include as much information as possible in your issue**. The more information you provide, the more likely your issue/ask will be understood and implemented. Helpful information includes:
+
+* What device you're running (inc. CPU type, memory, disk, etc.)
```
Comment (manodasanW) [inline CONTRIBUTING.md:30]: This might be useful in some cases, but I feel like the others are more important and maybe we should re-order it or add a caveat to this.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  ```
+
+  ... or Cmd:
+
+  ```cmd
+  C:\> ver
+
+  Microsoft Windows [Version 10.0.18900.1001]
+  ```
+
```
Comment (manodasanW) [inline CONTRIBUTING.md:57]: We should also ask about dotnet version, cswinrt version, sdk projection version.

**Comment by manodasanW** [comment]
Diagram_CreateProjection.jpg - we build both .net 5 version and .net standard 2.0 version of the projection.

File: README.md [hunk 3]
```diff
@@ -84,62 +42,29 @@ After a successful command-line build, the cswinrt.sln can be launched from the
...
+### License Info
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
+Most contributions require you to agree to a
 Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
 the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
 
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
```
Comment (manodasanW) [inline README.md:142]: We should probably keep this or a variant of it somewhere.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+If no existing item describes your issue/feature, great - please file a new issue:
+
+### File a new Issue
+
+* Don't know whether you're reporting an issue or requesting a feature? File an issue
+* Have a question that you don't see answered in docs, videos, etc.? File an issue
+* Want to know if we're planning on building a particular feature? File an issue
+* Got a great idea for a new utility or feature? File an issue/request/idea
+* Don't understand how to do something? File an issue/Community Guidance Request
```
Comment (Scottj1s) [inline CONTRIBUTING.md:23]: what's a Community Guidance Request ?  if not applicable, remove

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+* Found an existing issue that describes yours? Great - upvote and add additional commentary / info / repro-steps / etc.
+
+### Complete the template
```
Comment (Scottj1s) [inline CONTRIBUTING.md:26]: Do we have a new issue template (like Project Reunion does)?  I don't think so - maybe reword to "Include details" or something

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+If you don't have any additional info/context to add but would like to indicate that you're affected by the issue, upvote the original issue by clicking its [+😊] button and hitting 👍 (+1) icon. This way we can actually measure how impactful an issue is.
+
+---
+
+## Development
+
+### Fork, Clone, Branch and Create your PR
+
+Once you've discussed your proposed feature/fix/etc. with a team member, and you've agreed an approach or a spec has been written and approved, it's time to start development:
```
Comment (Scottj1s) [inline CONTRIBUTING.md:76]: agreed on an

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+1. Fork the repo if you haven't already
+1. Clone your fork locally
+1. Create & push a feature branch
+1. Create a [Draft Pull Request (PR)](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
+1. Work on your changes
+
+### Code Review
+
+When you'd like the team to take a look, (even if the work is not yet fully-complete), mark the PR as 'Ready For Review' so that the team can review your work and provide comments, suggestions, and request changes. It may take several cycles, but the end result will be solid, testable, conformant code that is safe for us to merge.
```
Comment (Scottj1s) [inline CONTRIBUTING.md:86]: fully complete (no hyphen)

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+## Development
+
+### Fork, Clone, Branch and Create your PR
+
+Once you've discussed your proposed feature/fix/etc. with a team member, and you've agreed an approach or a spec has been written and approved, it's time to start development:
+
+1. Fork the repo if you haven't already
+1. Clone your fork locally
+1. Create & push a feature branch
+1. Create a [Draft Pull Request (PR)](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
```
Comment (Scottj1s) [inline CONTRIBUTING.md:81]: We don't generally use draft PRs - why this guidance?

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+  C:\> ver
+
+  Microsoft Windows [Version 10.0.18900.1001]
+  ```
+
+* What tools and apps you're using (e.g. VS 2019, VSCode, etc.)
+* Don't assume we're experts in setting up YOUR environment and don't assume we are experts in YOUR workflow. Teach us to help you!
+* **We LOVE detailed repro steps!** What steps do we need to take to reproduce the issue? Assume we love to read repro steps. As much detail as you can stand is probably _barely_ enough detail for us!
+* Prefer error message text where possible or screenshots of errors if text cannot be captured
+* **If you intend to implement the fix/feature yourself then say so!** If you do not indicate otherwise we will assume that the issue is our to solve, or may label the issue as `Help-Wanted`.
```
Comment (Scottj1s) [inline CONTRIBUTING.md:62]: ours to solve

**Comment by Scottj1s** [comment]
Regarding the diagrams:
why is NETprojection italicized?
remove the netstandard2 comments in the diagrams? its confusing, since net5 is also supported

File: CONTRIBUTING.md (hunk 1 above)
Comment (angelazhangmsft) [reply CONTRIBUTING.md]: Reordered these and included cswinrt, SDK, .NET version

File: CONTRIBUTING.md (hunk 8 above)
Comment (angelazhangmsft) [reply CONTRIBUTING.md]: Changed to just a Pull Request
